### DISCUSSION TITLE
Is there a click function for clicking DOM elements?

### DISCUSSION THREAD

**Discussion by b1sar** [skrapeit/skrape.it#155] [Q&A] [answered]
I'm trying to scrape a dynamically rendered website and I need to click some buttons in order to get some data and without interacting with the website there is no way I can do what I want to do. I read the documentation and look around the source code but couldn't find a click function or something similar. Is there any interactive functionality  (like selenium's `WebElement::click()` or `sendKeys()` functions) in skrape.it?

**Comment by christian-draeger** [comment, marked as answer]
Currently not, sorry. Skrape{it} is just an html parser for now. With the BrowserFetcher It should be technically possible but is not implemented yet.

**Comment by b1sar** [comment]
Okay I got the point,  thanks for the answer.
